web/src/admin/policies/geoip/GeoIPPolicyForm.ts

```diff
@@ -79,13 +80,125 @@ export class GeoIPPolicyForm extends BasePolicyForm<GeoIPPolicy> {
                     )}
                 </p>
             </ak-form-element-horizontal>
-            <ak-form-group .expanded=${true}>
-                <span slot="header"> ${msg("Policy-specific settings")} </span>
+            <ak-form-group>
+                <span slot="header"> ${msg("Distance settings")} </span>
+                <div slot="body" class="pf-c-form">
+                    <ak-form-element-horizontal name="checkHistoryDistance">
+                        <label class="pf-c-switch">
+                            <input
+                                class="pf-c-switch__input"
+                                type="checkbox"
+                                ?checked=${this.instance?.checkHistoryDistance ?? false}
+                            />
+                            <span class="pf-c-switch__toggle">
+                                <span class="pf-c-switch__toggle-icon">
+                                    <i class="fas fa-check" aria-hidden="true"></i>
+                                </span>
+                            </span>
+                            <span class="pf-c-switch__label"
+                                >${msg("Check historical distance of logins")}</span
+                            >
+                        </label>
+                        <p class="pf-c-form__helper-text">
+                            ${msg(
+                                "When this option enabled, the GeoIP data of the policy request is compared to the specified number of historical logins.",
+                            )}
+                        </p>
+                    </ak-form-element-horizontal>
+                    <ak-form-element-horizontal
```

Comment:
No `ak-number-input` though, it doesn't support min yet. cc. @kensternberg-authentik for awareness

Reply:
https://github.com/goauthentik/authentik/pull/12703